docs/tasks/administer-cluster/cluster-management.md

```diff
@@ -23,6 +23,11 @@ The current state of cluster upgrades is provider dependent, and some releases m
 
 * [Upgrading to 1.6](/docs/admin/upgrade-1-6)
```

Comment:
This link is invalid. IMO this line is no longer needed. @brendandburns WDYT? If you agree, can you please remove it since you are here? :)